api/v2/endpoints/post-identity-map.md

```diff
@@ -142,3 +190,12 @@ The following table lists the `status` property values and their HTTP status cod
 | `unauthorized` | 401 | The request did not include a bearer token, included an invalid bearer token, or included a bearer token unauthorized to perform the requested operation. |
 
 If the `status` value is other than `success`, the `message` field provides additional information about the issue.
+
+### Identity Map Policy
+
+The identity map policy let the caller decide when to generate a token. It is passed as an **integer ID** in the request body (with key 'policy') If the parameter is omitted, policy with ID = 0 will be applied.
```

Comment:
The identity map policy lets the caller decide when to generate a token. It is passed as an **integer ID** in the request body (with key `policy`). If this parameter is not included, the default value, policy=1, is applied.

Reply:
great, changed!